JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx

```diff
@@ -1,44 +1,52 @@
 import { UploadFile as UploadFileIcon } from '@mui/icons-material'
-import { Box, CardActions, Container, Typography } from '@mui/material'
+import {
+  Box,
+  CardActions,
+  Checkbox,
+  Container,
+  FormControl,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography
+} from '@mui/material'
 import { useMutation } from '@tanstack/react-query'
 import { AxiosError, AxiosProgressEvent, AxiosRequestConfig } from 'axios'
 import { useSnackbar } from 'notistack'
-import { FC, useState } from 'react'
+import { FC, useRef, useState } from 'react'
 import { FileRejection, useDropzone } from 'react-dropzone'
 import ApiClient from '../../services/ApiClient'
 import { FileObj, UploadStatus } from '../../types/FileUpload'
 import Button from '../shared/Button'
 import './Import.css'
 import UploadFileListItem from './UploadFileListItem'
 
+const MAX_UPLOAD_FILE_SIZE_IN_BYTES =
+  process.env.REACT_APP_MAX_UPLOAD_CSV_SIZE_IN_BYTES ?? 128 * 1024 * 1024
```

Comment:
If there any particular reason we are doing the multiplication here, instead of just using the bytes. We can always leave a comment saying is 128MB for readability

Reply:
the comment could show different information than the actual value that is not human readable in bytes, showing file max size like that will ensure that the correct and right value is used.

Reply:
Okay. Thanks!

Reply:
Okay. Thanks!